<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgEditLevel</class>
 <widget class="QDialog" name="CDlgEditLevel">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>462</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>462</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>283</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Information</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hint / description</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="hint">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>59</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Additional information about this level</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="title">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>15</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Title of the level</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>186</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>UUID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eUUID">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>This is the unique identifier that for this level. It is used to reference this object.</string>
     </property>
     <property name="whatsThis">
      <string>This is the unique identifier that for this level. It is used to reference this object.</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Copy to clipboard</string>
     </property>
     <property name="whatsThis">
      <string>Copy to clipboard</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="lgck-builder.qrc">
       <normaloff>:/images/pd/copy.png</normaloff>:/images/pd/copy.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Colors</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Border Color</string>
     </property>
    </widget>
    <widget class="CWBtnColor" name="btnBorderColor">
     <property name="geometry">
      <rect>
       <x>282</x>
       <y>70</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="eBorderColor">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Border color of the level.</string>
     </property>
     <property name="inputMask">
      <string>hhhhhh</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Background color</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eIBkColor">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>160</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Background color for the transition screen</string>
     </property>
     <property name="inputMask">
      <string>hhhhhh</string>
     </property>
    </widget>
    <widget class="CWBtnColor" name="btnIBkColor">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>160</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="eITextColor">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>190</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Text color for the transition screen</string>
     </property>
     <property name="inputMask">
      <string>hhhhhh</string>
     </property>
    </widget>
    <widget class="CWBtnColor" name="btnITextColor">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Text Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Background color</string>
     </property>
    </widget>
    <widget class="CWBtnColor" name="btnColor">
     <property name="geometry">
      <rect>
       <x>282</x>
       <y>40</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="bkColor">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Background color of the level.</string>
     </property>
     <property name="inputMask">
      <string>hhhhhh</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>10</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Level&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Introduction&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eColorMod">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Primary light source for this level.</string>
     </property>
     <property name="inputMask">
      <string>hhhhhh</string>
     </property>
    </widget>
    <widget class="CWBtnColor" name="btnColorMod">
     <property name="geometry">
      <rect>
       <x>282</x>
       <y>100</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Color Mod</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Features</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>52</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Music</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eMusic">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>16</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Name of the music file to be played when the level start. This will loop until stopped.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eTimeLimit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Time to complete this level. (seconds)</string>
     </property>
     <property name="inputMask">
      <string>999</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>66</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Time Limit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eTickRate">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Game speed on this level.</string>
     </property>
     <property name="inputMask">
      <string>999</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>66</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eClosure">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Time until the level is closed. This is used to allow animation sequences to run until completion.</string>
     </property>
     <property name="inputMask">
      <string>9999</string>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>106</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Time to Close</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eTimeIntro">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>144</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Time until the level is closed. This is used to allow animation sequences to run until completion.</string>
     </property>
     <property name="inputMask">
      <string>9999</string>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Intro delay</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QComboBox" name="cbEndLevel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>13</y>
       <width>281</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>19</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Level</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTrigger">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>58</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>62</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fire trigger</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wrap arround</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cWrapUp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>top</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cWrapDown">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>bottom</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cWrapLeft">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>176</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>left</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cWrapRight">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>176</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>right</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cNoGravity">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Disable gravity on this level.</string>
     </property>
     <property name="text">
      <string>no gravity</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cOpenTop">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>open top</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>103</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>These options control whether the borders are will allow sprites to move from one to the other.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Events</string>
    </attribute>
    <widget class="QTreeWidget" name="treeEvents">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>221</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>This is a list of all the events available for this level.
Double-click on the one you want to edit.</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Canvas</string>
    </attribute>
    <widget class="QCheckBox" name="cCanvas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Set canvas size (experimental)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>95</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eWidth">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>55</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="eHeight">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>91</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnWorldSize">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>93</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CWBtnColor</class>
   <extends>QPushButton</extends>
   <header>wbtncolor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="lgck-builder.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDlgEditLevel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDlgEditLevel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
